etesync-server.ini.example

```diff
@@ -1,5 +1,9 @@
 [global]
 secret_file = secret.txt
+static_root = /path/to/static
```

Comment:
Could you please have these 4 new ones commented out by default with a note that they are optional and you probably don't want to change them unless you know what you're doing?